modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -641,10 +657,14 @@ void ONNXCompiled::Run(const std::vector<cv::Mat>& ins,
                          &out_tensors.front(),
                          params.output_names.size());
     } else {
+        GAPI_Assert(is_dynamic || is_postproc);
```

Comment:
Isn't it always true? It's just after condition

Reply:
No

Reply:
But.. it's just after the `if (!is_dynamic && !is_postproc)` condition...

Reply:
Removed.